lib/post/bloc/post_bloc.dart

```diff
@@ -554,6 +560,32 @@ class PostBloc extends Bloc<PostEvent, PostState> {
     }
   }
 
+  Future<void> _reportCommentEvent(ReportCommentEvent event, Emitter<PostState> emit) async {
+    try {
+      emit(state.copyWith(status: PostStatus.refreshing, moddingCommentId: event.commentId, selectedCommentId: state.selectedCommentId, selectedCommentPath: state.selectedCommentPath));
+
+      Account? account = await fetchActiveProfileAccount();
+      LemmyApiV3 lemmy = LemmyClient.instance.lemmyApiV3;
+
+      if (account?.jwt == null) {
+        return emit(state.copyWith(
+            status: PostStatus.failure, errorMessage: 'You are not logged in. Cannot report a comment.', selectedCommentId: state.selectedCommentId, selectedCommentPath: state.selectedCommentPath));
```

Comment:
Can you localize this? For context, you can use `GlobalContext.context`.

Reply:
Sure

Reply:
Done